coreneuron/permute/cellorder.cu

```diff
@@ -0,0 +1,101 @@
+#ifdef ENABLE_CUDA_INTERFACE
+
```

Comment:
nitpick: to remove; we can compile cuda file in GPU build and use CLI option as mentioned before.